package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-advertising",
-  "version": "4.1.2",
+  "version": "4.2.0-beta.25",
```

Comment:
I think this will be updated automatically when we publish the new version.

Reply:
Then should we remove it before merging it right? To not cause any conflict.